controller/search_blackbox_test.go

```diff
@@ -403,6 +405,140 @@ func (s *searchControllerTestSuite) TestSearchFilter() {
 	assert.Equal(s.T(), "specialwordforsearch", r.Attributes[workitem.SystemTitle])
 }
 
+func (s *searchControllerTestSuite) TestSearchByHierarchy() {
```

Comment:
@kwk Why do you call this `SearchByHierarchy`? and not `SearchByTypeGroup`?

Reply:
Good question @DhritiShikhar . Let me re-work that.

Reply:
Meanwhile this has been renamed to `TestSearchByWorkItemTypeGroup`.